### DISCUSSION TITLE
How do I add a game to the Favorites in Onion?

### DISCUSSION THREAD

**Discussion by midnitefox** [OnionUI/Onion#550] [Q&A] [answered]
I've looked everywhere within Onion I think, and searched around online including the wiki, but can't find an answer.

**Comment by dwmccoy** [comment, marked as answer]
When you are browsing your game list, highlight the game game you want to add to the Favorites list and press the Menu Button.  Select Add as Favorite from there.  To remove a Favorite from the Favorite games list, Press the Menu Button and select  Remove.
